src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -775,19 +780,25 @@ private void packageTestResults() {
      * html file
      */
     private String generateImageLink(String imageName) {
-        String imageLink = "<br/>";
+        StringBuilder imageLink = new StringBuilder("<br/>");
         if (imageName.length() >= directory.length() + 1) {
-            imageLink += "<a href='javascript:void(0)' onclick='toggleImage(\"" +
-                    imageName.substring(directory.length() + 1) + "\")'>Toggle Screenshot Thumbnail</a>";
-            imageLink += " <a href='javascript:void(0)' onclick='displayImage(\"" +
-                    imageName.substring(directory.length() + 1) + "\")'>View Screenshot Fullscreen</a>";
-            imageLink += "<br/><img id='" + imageName.substring(directory.length() + 1) + "' border='1px' src='" +
-                    imageName.substring(directory.length() + 1) + "' width='" + EMBEDDED_IMAGE_WIDTH +
-                    "px' style='display:none;'></img>";
+            imageLink.append("<a href='javascript:void(0)' onclick='toggle(\"").
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "<a href='javascript:void(0)' onclick='toggle(\"" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)